bindings/matrix-sdk-crypto-nodejs/download-lib.js

```diff
@@ -21,15 +21,18 @@ function download_lib(libname) {
 
     const url = `${DOWNLOADS_BASE_URL}/${CURRENT_VERSION}/${libname}`;
     console.info(`Downloading lib ${libname} from ${url}`);
+
     const dl = new DownloaderHelper(url, __dirname, {
-        override: true,
-        httpsRequestOptions: {
-          // Disable keepalive to prevent the process hanging open.
-          // https://github.com/matrix-org/matrix-rust-sdk/issues/1160
-          agent: new Agent({ keepAlive: false }),
-        },
+      override: true,
     });
 
+    var proxy = process.env.https_proxy ?? process.env.http_proxy ?? undefined
```

Comment:
Why check for `http_proxy` as well? I think we should only use `https_proxy`. Also is there some magic going on or is `process.env.foo` just for accessing an environment variable called `foo`? If so, I think it makes sense to check for `HTTPS_PROXY` in addition. Finally, isn't `?? undefined` a no-op?

Reply:
> Why check for http_proxy as well? I think we should only use https_proxy. 

Good point, I'll remove it.

> Also is there some magic going on or is process.env.foo just for accessing an environment variable called foo? If so, I think it makes sense to check for HTTPS_PROXY in addition. 

I agree, I'll add a check for `HTTPS_PROXY` as well!

>  Finally, isn't ?? undefined a no-op?

Yes you are correct, I'll remove it.